File: 프로그래머스/lv1/12906. 같은 숫자는 싫어/같은 숫자는 싫어.cpp
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr) 
{
    vector<int> answer = arr;
    auto last = unique(answer.begin(), answer.end());
    answer.erase(last, answer.end());
    return answer;
}

File: 프로그래머스/1/12906. 같은 숫자는 싫어/같은 숫자는 싫어.py
# 배열 arr에서 연속적으로 나타나는 숫자는 하나만 남기고 전부 제거

def solution(arr):
    answer = []
    answer.append(arr[0])
    for i in range(1, len(arr)):
        if answer[-1] != arr[i]:
            answer.append(arr[i])
    return answer